isv.md

```diff
@@ -0,0 +1,46 @@
+| DevOps | [Harness (Drone.io)](https://partners.amazonaws.com/partners/0010L00001v2cHWQAY) | [Drone (Runner & Server)](https://docs.drone.io/runner/overview/) | Release: https://docs.drone.io/runner/docker/installation/linux/<br>Announcement: https://blog.drone.io/drone-announces-official-support-for-arm/ |
+| DevOps | [HashiCorp](https://partners.amazonaws.com/partners/001E000001BxMyKIAV) | [Terraform](https://www.hashicorp.com/products/terraform) | Release: https://www.terraform.io/downloads.html<br>Announcement: https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-14-beta |
+| DevOps | [HashiCorp](https://partners.amazonaws.com/partners/001E000001BxMyKIAV) | [Vault](https://www.hashicorp.com/products/vault) | Release: https://www.vaultproject.io/downloads<br>Announcement: https://github.com/hashicorp/vault/milestone/17?closed=1 |
+| DevOps | [Progress (Chef)](https://partners.amazonaws.com/partners/001E000000Rp5GkIAJ) | [Chef Infra Client](https://www.chef.io/products/chef-infra) | Release: https://downloads.chef.io/products/infra-client<br>Announcement: https://www.chef.io/press-release/chef-infra-16-empowers-experts-and-simplifies-adoption-for-new-users |
+| DevOps | [Puppet](https://partners.amazonaws.com/partners/001E000000Rl12xIAB) | [Puppet Enterprise: Agent](https://puppet.com/products/puppet-enterprise/) | Release: https://puppet.com/docs/pe/latest/supported_operating_systems.html<br>Announcement: https://puppet.com/blog/puppet-enterprise-adds-arm-high-sierra-support-ssl-curl-updates/ |
+| DevOps | [Teleport (Gravitational)](  https://goteleport.com/) | [Teleport Unified Access Plane](https://goteleport.com/teleport/) | Release: https://goteleport.com/teleport/download/<br>Announcement: https://goteleport.com/teleport/docs/changelog/#500 |
+| DevOps | [Travis CI](https://partners.amazonaws.com/partners/001E0000019F9EUIA0) | [Travis CI (SaaS)](https://travis-ci.com/) | Release: "arch: arm64-graviton2" https://docs.travis-ci.com/user/multi-cpu-architectures/<br>Announcement: https://blog.travis-ci.com/2020-09-11-arm-on-aws |
+| Networking | [F5 (NGINX)](https://partners.amazonaws.com/partners/001E000000Rl12PIAR) | [NGINX Plus](https://www.nginx.com/products/nginx/) | Release: https://aws.amazon.com/marketplace/pp/Nginx-Software-Inc-NGINX-Plus-Enterprise-Amazon-Li/B07RQN9ZQP<br>Announcement: https://www.prnewswire.com/news-releases/nginx-plus-now-available-for-latest-generation-of-arm-based-64-bit-servers-665653129.html |
+| Observability | [Datadog](https://partners.amazonaws.com/partners/001E000000Rp57sIAB) | [Datadog Agent](https://docs.datadoghq.com/agent/) | Release: https://hub.docker.com/r/datadog/agent<br>Announcement: https://www.datadoghq.com/blog/datadog-arm-agent/ |
```

Comment:
is Observability or Monitoring a better fit ?

Reply:
I wondered the same too and learned that the line is blurring for many products as they gain more capabilities. "Observability" as a broad category covers the agent/tools on this list today.